lib/index.js

```diff
@@ -81,26 +84,28 @@ export class Kadira {
   }
 
   // send the given payload to the server
-  sendData(_payload) {
+  async sendData(_payload) {
```

Comment:
This `async` doesn't seem to be needed.

Reply:
I added it so it always returns a promise as is expected.

Reply:
Mostly due to the throw nested in it... it might work without, will check.

Reply:
It did, but I had to surround it with a Promise.